hazelcast/src/test/java/com/hazelcast/test/annotation/LiveTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+
+/**
+ * Annotates tests which requires integration tests over a live api.
+ * <p/>
+ * Will be executed in nightly builds and for jclouds live tests
+ */
+public final class LiveTest {
```

Comment:
I made a separate profile for integration tests. Our coverage profile will no longer be dependent to jclouds. We might run integration test weekly for example we mostly cover our jclouds classes with unit tests and maybe run jclouds test weekly with  `mvn clean install -Pjclouds-test -Dtest.aws-ec2.identity="<aws-key>" -Dtest.aws-ec2.credential="<aws-credential>"` on seperate build on cloudbeees.